src/common/components/Icons/CheckIcon.tsx

```diff
@@ -17,6 +19,9 @@ export const CheckIcon = ({
       fill='none'
       xmlns='http://www.w3.org/2000/svg'
       className={className}
+      onClick={() => {
```

Comment:
Could you please remove onClick from icons at all?

The point is: some users are using Tab to navigate faster, and this won't work using svg with onClick.
Developer should wrap icon with `<button onClick={onClick}>` tag and all will be fine.

Reply:
Okay, will remove onClick function for all svg icons.

Reply:
Can't see removal, perhaps forgot to push?